### ISSUE TITLE
A little confused about 'prep_unet'

### ISSUE THREAD

**Issue by yangkunl** [HyelinNAM/ContrastiveDenoisingScore#9]
Hi,congratulations on CVPR24! I have a question.Regarding the setting of `params.requires_grad = True`  in the  [prep_unet function](https://github.com/HyelinNAM/ContrastiveDenoisingScore/blob/9cc49b9ebebfabaf78f81944db7efb0a1affa01f/utils/attention.py#L38) Is Unet being trained?

**Comment by HyelinNAM** [comment]
Hi,

CDS doesn't required training. 
I guess I just borrowed `prep_unet` function from other repo. 
On second thought, they are not essential since gradients calculation automatically happen.

Best,
Hyelin
